build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         classpath "com.android.tools.build:gradle:${ANDROID_GRADLE_PLUGIN_VERSION}"
         classpath "org.robolectric:robolectric-gradle-plugin:${ROBOLECTRIC_GRADLE_PLUGIN_VERSION}"
         classpath "com.jakewharton.sdkmanager:gradle-plugin:${SDKMANAGER_GRADLE_PLUGIN_VERSION}"
-
+	classpath "com.github.dcendents:android-maven-gradle-plugin:1.3"
```

Comment:
I guess you have used tabs instead of spaces here, that is why diff looks not aligned.
Please use spaces instead.